### ISSUE TITLE
Multi-tenant: Sites, companies?

### ISSUE THREAD

**Issue by reillychase** [ubiquiti/eot-controller#5]
I know you can group devices, but ideally, there would be 3 layers:

1. Device Groups
2. Sites
3. Companies

**Comment by yen-ubnt** [comment]
correct, it will require cloud access and cloud site. We will implement it later.

**Comment by reillychase** [comment]
I don't like the idea of cloud access being the solution to multi-tenant, that would still require an instance of UniFi LED at every single location.

Consider this:

- 1 UniFi LED server on a cloud VPS
- Many devices connect directly to it

In that case it would be good to be able to group the devices per location and even per company.

Right now UniFi controller lacks some multit-tenant ability too because it has the concept of sites but not companies. One big problem that creates is you can't give an Admin access to more than one site in UniFi

**Comment by reillychase** [comment]
Just following up to see if Ubiquiti plans to multi-tenant EoT. It would be great for installers to manage multiple customers from 1 cloud server.

**Comment by yen-ubnt** [comment]
Yes, cloud access will be support in the future

Best regards,

Yen Tung
Ubiquiti Networks
T: 07010160020#644 (Taiwan)
M: 0932390784 (Taiwan)
M: 415-988-0824 (Taiwan)

**Comment by reillychase** [comment]
“Cloud access” and “cloud server” are very different. Cloud access allows you to manage a local UniFi EoT server while away, but what I am asking is about having a cloud server for devices from multiple locations to be centrally connected and managed on a remote server on DigitalOcean, or AWS etc.

I hope this will be a priority so that installers can centrally manage a single server instead of dozens of cloud keys.

It would require major changes to the current code because it hasn’t been built so far to support more than one location, and no layer 3 adoption options.
